tools/metaquantome/metaquantome_viz.xml

```diff
@@ -90,7 +90,36 @@
         #if $plot.height:
             --height=$plot.height
         #end if
-        --img='outfile.png'
+        --img='outfile.png' &&
+        mkdir '${ wrapped_outfile.files_path }' &&
+        cp outfile.png '${ wrapped_outfile.files_path }' &&
+        ## stylesheet
+        echo "img {
+          display: block;
+          margin-left: auto;
+          margin-right: auto;
+          max-width: 75%;
+          height: auto;
+        }" > style.css &&
+        cp style.css '${ wrapped_outfile.files_path }' &&
+        echo "<!DOCTYPE html>
+        <html>
+          <head>
+            <link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\"/>
+          </head>
+          <body>
+          <h1>Output Figure</h1>
+          <br>
+            <img src=\"outfile.png\" width=600 height=600>
+            <br>
+            <p>To download the image, right-click and choose \"Save image as...\"</p>
+            <br>
+            <p>OR</p>
+            <br>
+            <p><a href=\"outfile.png\">Right Click Here</a></p>
+          </body>
+        </html>" > wrapped_outfile.html &&
+        cat wrapped_outfile.html
```

Comment:
is that cat still needed?

Reply:
Thanks for pointing this out. "cat" not required.

Reply:
Updated the code.